src/functionalTest/java/uk/gov/hmcts/reform/pbis/e2e/EndToEndTest.java

```diff
@@ -87,9 +87,12 @@ public void should_send_welcome_email_when_valid_message_present() throws Except
 
         softly.assertThat(email.getNotificationType()).isEqualTo("email");
         softly.assertThat(email.getTemplateId().toString()).isEqualTo(config.getTemplateId());
-        softly.assertThat(email.getBody()).contains("First name: " + registration.firstName);
-        softly.assertThat(email.getBody()).contains("Last name: " + registration.lastName);
-        softly.assertThat(email.getBody()).contains("Welcome link: " + config.getWelcomeLink());
+
+        // check email template was filled with appropriate values
+        softly.assertThat(email.getBody()).contains(registration.firstName);
+        softly.assertThat(email.getBody()).contains(registration.lastName);
+        softly.assertThat(email.getBody()).contains(config.getWelcomeLink());
```

Comment:
used to rely on very specific template existing in gov notify. now just check that the values were passed to the template